### PR TITLE
tidb-binlog: add binlog-control.md

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

Add binlog-control.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3347
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
Please help review this PR. @CaitinChen

File: tidb-binlog/binlog-control.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+-data-dir string
+    The file path where the checkpoint file of Drainer is stored ("binlog_position" by default) (deprecated)
+-node-id string
+    The ID of Pump or Drainer
+-pd-urls string
+    The address of PD. If multiple addresses exist, use "," to separate each ("http://127.0.0.1:2379" by default)
+-ssl-ca string
+    The file path of SSL CAs
+-ssl-cert string
+    The file path of the X509 certificate file in the PEM format
```
Comment (CaitinChen) [inline tidb-binlog/binlog-control.md:76]: Is this line's indentation correct? It is not the same as that in Chinese.
+@WangXiangUSTC

File: tidb-binlog/binlog-control.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+## Descriptions
+
+Command line parameters:
+
+```
+Usage of binlogctl:
+-V
+    Outputs the binlogctl version information
```
[the remark this answers is not in the document]
Comment (CaitinChen) [reply tidb-binlog/binlog-control.md:64]: Is this line's indentation correct? It is not the same as that in Chinese.
+@WangXiangUSTC

File: tidb-binlog/binlog-control.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+-node-id string
+    The ID of Pump or Drainer
+-pd-urls string
+    The address of PD. If multiple addresses exist, use "," to separate each ("http://127.0.0.1:2379" by default)
+-ssl-ca string
+    The file path of SSL CAs
+-ssl-cert string
+    The file path of the X509 certificate file in the PEM format
+-ssl-key string
+    The file path of X509 key file of the PEM format
```
[the remark this answers is not in the document]
Comment (CaitinChen) [reply tidb-binlog/binlog-control.md:78]: Is this line's indentation correct? It is not the same as that in Chinese.
+@WangXiangUSTC

File: tidb-binlog/binlog-control.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    If a time zone is set, the corresponding time of the obtained `TSO` is printed in the "generate_meta" mode. For example, "Asia/Shanghai" is the CST time zone and "Local" is the local time zone
+-show-offline-nodes
+    Used with the `-cmd pumps` or `-cmd drainers` command. The two commands do not show the offline node by default unless this parameter is explicitly specified
+```
+
+Command examples:
+
+- Check the state of all the Pump or Drainer nodes.
+
+    Set `cmd` as `pumps` or `drainers`. For example:
```
Comment (CaitinChen) [inline tidb-binlog/binlog-control.md:89]:
```suggestion
    Set `cmd` to `pumps` or `drainers`. For example:
```

File: tidb-binlog/binlog-control.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+    ```
+    [2019/04/28 09:29:59.016 +00:00] [INFO] [nodes.go:48] ["query node"] [type=drainer] [node="{NodeID: 1.1.1.1:8249, Addr: 1.1.1.1:8249, State: online, MaxCommitTS: 408012403141509121, UpdateTime: 2019-04-28 09:29:57 +0000 UTC}"]
+    ```
+
+- Pause or close Pump or Drainer.
+
+    You can use the following commands to pause or close services:
+
+    | cmd             | Description           | Example                                                                                             |
```
Comment (CaitinChen) [inline tidb-binlog/binlog-control.md:115]:
```suggestion
    | Command             | Description           | Example                                                                                             |
```

File: tidb-binlog/binlog-control.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+    | pause-pump      | Pause Pump      | `bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd pause-pump -node-id ip-127-0-0-1:8250`       |
+    | pause-drainer   | Pause Drainer   | `bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd pause-drainer -node-id ip-127-0-0-1:8249`    |
+    | offline-pump    | Close Pump      | `bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd offline-pump -node-id ip-127-0-0-1:8250`     |
+    | offline-drainer | Close Drainer   | `bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd offline-drainer -node-id ip-127-0-0-1:8249`  |
+
+    `binlogctl` sends the HTTP request to the Pump or Drainer node. After receiving the request, the node executes the exiting procedures accordingly.
+
+- Modify the state of a Pump or Drainer node in abnormal states.
+
+    When a Pump or Drainer node runs normally or when it is paused or closed in the normal process, it is in the right state. In abnormal states, the Pump or Drainer node cannot correctly maintain its state. This affects data replication tasks. In this case, use `binlogctl` to repair the state information.
```
Comment (CaitinChen) [inline tidb-binlog/binlog-control.md:126]:
```suggestion
    When a Pump or Drainer node runs normally or when it is paused or closed in the normal process, it is in the normal state. In abnormal states, the Pump or Drainer node cannot correctly maintain its state. This affects data replication tasks. In this case, use `binlogctl` to repair the state information.
```

File: tidb-binlog/binlog-control.md (hunk 5 above)
Comment (CaitinChen) [reply tidb-binlog/binlog-control.md:115]: Or `cmd`

File: tidb-binlog/binlog-control.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd update-pump -node-id ip-127-0-0-1:8250 -state paused
+    ```
+
+    > **Note:**
+    >
+    > When a Pump or Drainer node runs normally, it regularly updates its state to PD. The above command is used to directly modify the Pump or Drainer state saved in PD; therefore, do not use the command when the Pump or Drainer node runs normally. For more information, refer to [TiDB Binlog FAQ](/tidb-binlog/tidb-binlog-faq.md).
```
Comment (CaitinChen) [inline tidb-binlog/binlog-control.md:138]:
```suggestion
    > When a Pump or Drainer node runs normally, it regularly updates its state to PD. The above command directly modifies the Pump or Drainer state saved in PD; therefore, do not use the command when the Pump or Drainer node runs normally. For more information, refer to [TiDB Binlog FAQ](/tidb-binlog/tidb-binlog-faq.md).
```

**Comment by toutdesuite** [comment]
@CaitinChen PTAL again, thanks

**Pull request merged**